### PR TITLE
Generate indexes on basic columns and relate columns. Add kadastraleo…

### PR DESCRIPTION
**Pull request by JKoetsier**
…bjecten relations to sources

### CODE REVIEW

File: gobcore/model/sa/gob.py [hunk 1]
```diff
@@ -29,6 +34,25 @@ def get_column(column_name, column_specification):
     return column
 
 
+def _fields_for_collection(collection):
```
Comment (Vegter) [inline gobcore/model/sa/gob.py]: What is the difference between this one model['all_fields']
model['all_fields'] is defined in GOBModel.__init__

File: gobcore/model/sa/gob.py [hunk 2]
```diff
@@ -76,29 +98,100 @@ def columns_to_model(table_name, columns):
             # }
 
             # the GOB model for the specified entity
-            fields = {col: desc for col, desc in collection['fields'].items()}
+            entity = _fields_for_collection(collection)
+
```
Comment (Vegter) [inline gobcore/model/sa/gob.py]: entity = model.get_collection(cataloq_name, collection_name)['all_fields'])

File: gobcore/model/sa/gob.py [hunk 3]
```diff
@@ -76,29 +98,100 @@ def columns_to_model(table_name, columns):
...
+
+def _remove_leading_underscore(s: str):
+    return re.sub(r'^_', '', s)
+
 
-            state_fields = STATE_FIELDS if collection.get('has_states') else {}
+def _tablename_from_ref(ref: str):
+    catalog, collection = ref.split(':')
+    return model.get_table_name(catalog, collection)
```
Comment (Vegter) [inline gobcore/model/sa/gob.py]: Can't we add this method to GOBModel?
The code to split a ref is now duplicated at many places. By placing this code in GOBModel we can prevent further multiplication

File: gobcore/model/sa/gob.py (hunk 1 above)
Comment (JKoetsier) [reply gobcore/model/sa/gob.py]: Good catch. Does the exact same thing. Removed _fields_for_collection.

File: gobcore/model/sa/gob.py (hunk 3 above)
Comment (JKoetsier) [reply gobcore/model/sa/gob.py]: Done :)

**Pull request merged**